tests/xtimer_drift/main.c

```diff
@@ -108,9 +112,12 @@ void *worker_thread(void *arg)
         hr = sec / 3600;
         if ((loop_counter % TEST_HZ) == 0) {
             uint32_t expected = start + loop_counter * TEST_INTERVAL;
-            int32_t diff = now - expected;
-            printf("now=%" PRIu32 ".%06" PRIu32 " (%u hours %u min), diff=%" PRId32 "\n",
-                sec, us, hr, min, diff);
+            int32_t drift = now - expected;
+            expected = last + TEST_HZ * TEST_INTERVAL;
```

Comment:
No, the compiler will calculate the product `TEST_HZ * TEST_INTERVAL` in advance and just write that value in the machine code.